logstash-filter-http.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   # Gem dependencies
   s.add_runtime_dependency 'logstash-core-plugin-api', '>= 1.60', '<= 2.99'
   s.add_runtime_dependency 'logstash-mixin-ecs_compatibility_support', '~>1.2'
-  s.add_runtime_dependency "logstash-mixin-http_client", ">= 7.1.0", '< 9.0.0'
+  s.add_runtime_dependency "logstash-mixin-http_client", ">= 7.2.0", '< 9.0.0'
```

Comment:
The http_poller plugin doens't have the `< 9.0.0` to you think we could remove it?

Reply:
this is a common pattern ruby gems use to limit using a later version of the library, 
you know there's a current 7.x versions that your code works with, 8.x will probably work as well but another major might be a bit too optimistic of an assumption - thus the upper bound limit.